src/components/MainPage/ListingComponents/Grid.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div class="mainGrid">
+        <div class="outlineGrid" v-for="entry in entries" :key="entry.id">
+            <div class="contentGrid">
+                <div class="headGrid">
+                    <div class="titleGrid">{{entry.job}}</div>                    
+                    <div class="experienceGrid">Experience: <span style="fontWeight: 500">{{entry.experience}}</span></div>                    
+                </div>          
+                <div class="careerGrid">
+                    <div class="careerSub">🏠 {{entry.location}}</div>                
+                    <div class="careerSub">💼 {{entry.industry}}</div>                
+                </div>      
+                <div class="moneySct">                    
+                    <div class="moneyGrid">Earnings: <span style="fontWeight: 500">{{entry.earnings}}</span></div>                    
+                    <div class="moneyGrid">Expenses: <span style="fontWeight: 500">{{entry.expenses}}</span></div>
+                </div>       
+                <div class="textGrid">“{{entry.quote}}“</div>         
+                <div class="tagsGrid">
+                    <div class="tag1">{{entry.tag1}}</div>
+                    <div class="tag2">{{entry.tag2}}</div>
+                </div>
+                <div class="dateGrid">{{entry.date}}</div>
+            </div>            
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { mapState } from 'vuex'
+
+@Component({        
+    computed: {
+        ...mapState({
+            // eslint-disable-next-line
+            entries: (state: any) => state.listings.entries
```

Comment:
Same here

Reply:
All changes have been made.
changes:
- interface for modules (actions, getters, states)
- removed eslint-disabled-next-line
- integrated vuex-class
plz npm install for new dependencies